frontend/src/components/Utils/time.test.js

```diff
@@ -34,4 +34,22 @@ test('other time timestamp format', () => {
   const actualSingapore = utils.timestampToTimeFormatted(testDate, 'Asia/Singapore');
   expect(actualCentral).toEqual(expectedCentral);
   expect(actualSingapore).toEqual(expectedSingapore);
-})
\ No newline at end of file
+})
+
+test('new york full timestamp format', () => {
+  // Month parameter is zero indexed so it's actually the 10th month.
+  const testDate = new Date(Date.UTC(2020, 9, 3, 14, 19, 4, 23)).getTime();
+  const expected = "Saturday, October 3, 2020, 10:19 AM";
+  const actual = utils.timestampToFormatted(testDate);
+  expect(actual).toEqual(expected);
+});
+
+test('other full timestamp format', () => {
+  const testDate = new Date(Date.UTC(2020, 7, 23, 2, 3, 2, 4)).getTime();
+  const expectedCentral = "Saturday, August 22, 2020, 9:03 PM";
+  const expectedSingapore = "Sunday, August 23, 2020, 10:03 AM";
+  const actualCentral = utils.timestampToFormatted(testDate, "America/Chicago");
+  const actualSingapore = utils.timestampToFormatted(testDate, "Asia/Singapore");
```

Comment:
Maybe can make these top level constants? I think the time zones at the least could be put as constants.